_includes/header.html

```diff
@@ -71,9 +71,9 @@
         <div class="col-sm-12 col-md-3" style="margin-top: 12px">
           <form action="{{site.baseurl}}/search/" role="form" method="get">
             <div class="input-group input-group-sm">
-              <input type="text" class="form-control" name="q" placeholder="Search ROS packages">
+              <input type="text" class="form-control" name="q" placeholder="Find in rosindex">
               <span class="input-group-btn">
-                <button class="btn btn-default btn-sm" type="submit"><span class="glyphicon glyphicon-search"></span></button>
+                <button class="btn btn-default btn-sm" type="submit"><span class="glyphicon glyphicon-search" title="Find in rosindex"></span></button>
```

Comment:
Same here about moving the title to `<button>`

Reply:
Done, thanks!